### PR TITLE
add sample config for pyroscope

### PR DESCRIPTION
**Pull request by QuantumEnigmaa**
Towards https://github.com/giantswarm/giantswarm/issues/29151

### CODE REVIEW

**Review by QuentinBisson — APPROVED** [review]
Damn that's complex 😅

File: sample_configs/values-pyroscope-capz.yaml [hunk 1]
```diff
@@ -0,0 +1,825 @@
...
+        // ```
+        // profiles.grafana.com/memory.scrape: "true"
+        // profiles.grafana.com/memory.port: "8080"
+        // profiles.grafana.com/cpu.scrape: "true"
+        // profiles.grafana.com/cpu.port: "8080"
+        // profiles.grafana.com/goroutine.scrape: "true"
+        // profiles.grafana.com/goroutine.port: "8080"
+        // ```
+        //
+        // will scrape the `memory`, `cpu` and `goroutine` profiles from the `8080` port of the pod.
```
Comment (TheoBrigitte) [inline sample_configs/values-pyroscope-capz.yaml:59]: Add a note about default port naming https://grafana.com/docs/pyroscope/latest/deploy-kubernetes/helm/#optional-scrape-your-own-workloads-profiles

File: sample_configs/values-pyroscope-capz.yaml [hunk 2]
```diff
@@ -0,0 +1,825 @@
...
+        //
+        // For more information see https://grafana.com/docs/phlare/latest/operators-guide/deploy-kubernetes/#optional-scrape-your-own-workloads-profiles
```
Comment (TheoBrigitte) [inline sample_configs/values-pyroscope-capz.yaml:61]: Replace link with this
https://grafana.com/docs/pyroscope/latest/deploy-kubernetes/helm/#optional-scrape-your-own-workloads-profiles

Comment (QuantumEnigmaa) [reply sample_configs/values-pyroscope-capz.yaml:61]: Done upstream :)

File: sample_configs/values-pyroscope-capz.yaml (hunk 1 above)
Comment (QuantumEnigmaa) [reply sample_configs/values-pyroscope-capz.yaml:59]: Added in this PR : https://github.com/giantswarm/grafana-agent-app/pull/39

**Pull request merged**